query-parser/tests/find_tests.rs

```diff
@@ -208,3 +210,27 @@ fn can_parse_simple_or_and_join() {
                    )),
                ]);
 }
+
+#[test]
+fn can_parse_order_by() {
+    let invalid = "[:find ?x :where [?x :foo/baz ?y] :order]";
+    assert!(parse_find_string(invalid).is_err());
+
+    // Defaults to ascending.
+    let default = "[:find ?x :where [?x :foo/baz ?y] :order ?y]";
+    assert_eq!(parse_find_string(default).unwrap().order,
+               Some(vec![Order(Direction::Ascending, Variable::from_valid_name("?y"))]));
+
+    let ascending = "[:find ?x :where [?x :foo/baz ?y] :order (asc ?y)]";
```

Comment:
I find the direction first awkward, especially since SQL is `ORDER BY variable ASC`.  Why not follow suit?